tests/test_travis_mf_multiplePS/Makefile.am

```diff
@@ -0,0 +1,71 @@
+AUTOMAKE_OPTIONS = foreign
```

Comment:
multiplePS -> test_travis_mf_multiple_ps

Reply:
Fixed

Reply:
I assume this means you wanted it to be renamed.